marmoset/installimage/installimage_config.py

```diff
@@ -57,9 +61,9 @@ def __read_config_file(self, path=None):
 
         lines = []
 
-        with open(path, 'r') as f:
-            lines = f.readlines()
-            f.close()
+        with open(path, 'r') as file:
+            lines = file.readlines()
+            file.close()
```

Comment:
when you use the "with open" function, you do not have to clean the file explicit:
https://docs.python.org/3.5/tutorial/inputoutput.html#methods-of-file-objects (last example of that section)